### PR TITLE
Phoebemirman/ch12489/create integration test for account walkthrough

### PR DESCRIPTION
**Pull request by phoebemirman**

### CODE REVIEW

**Comment by adrake33** [comment]
This pull request **introduces 1 alert** when merging d1e0dfbd3747a337cb569bdf96ae314c1640f7c8 into 225c3f6299342e3bfb9f8e355e55e55f603fe538 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur/rev/pr-4a2616017276a399206acce881e3f18eae1f29c6)

**new alerts:**

* 1 for Syntax error

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by adrake33** [comment]
This pull request **introduces 1 alert** when merging 625e729e19006196484fe95824fdf58ebae37357 into 225c3f6299342e3bfb9f8e355e55e55f603fe538 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur/rev/pr-afdbb3be7890b5800607d8365074f1a1b117b736)

**new alerts:**

* 1 for Syntax error

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: types/window.d.ts [hunk 1]
```diff
@@ -2,6 +2,10 @@ interface IntegrationHelpers {
   updateAccountAddress(address: string): void;
   findMarketId(marketDescription: string): void;
   hasDisclaimerModalBeenDismissed(): boolean;
+  logout(): void;
+  getAccountData(): void;
+  formatRep(value: string): void;
+  formatEth(value: string): void;
 }
```
Comment (bthaile) [inline types/window.d.ts]: shouldn't `formatRep` and `formatEth` return object?
better would be a formatted object

**Comment by adrake33** [comment]
This pull request **introduces 1 alert** when merging cfccd9c2a9413a846500addd2138281d92fe06bb into d2f27646ebe4ea4d32587f9501a62bf67b31d184 - [view on LGTM.com](https://lgtm.com/projects/g/AugurProject/augur/rev/pr-89ebbb759c18f7f41b879dd152f950a2ae32f15c)

**new alerts:**

* 1 for Syntax error

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**